docs/Reference/README.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: default
+title: "Reference"
+---
+
+These resources act as references that provide technical descriptions of F'. You will find API documentation here.
+
+## API Documentation
+1. [GDS CLI Design](./fprime/UsersGuide/dev/gds-cli-dev.html)
+2. [C++ Documentation](./fprime/UsersGuide/api/c++/html/index.html)
+3. [CMake User API](./fprime/UsersGuide/cmake/cmake-api.html)
```

Comment:
These 2 links should point to the actual URLs
https://nasa.github.io/fprime/UsersGuide/api/c++/html/index.html
https://nasa.github.io/fprime/UsersGuide/cmake/cmake-api.html

Reply:
I updated the link in the TOC to point to the URL you provided.